apps/nuxt3-ssr/components/ResourceCard.vue

```diff
@@ -15,6 +15,9 @@ const props = withDefaults(
   }
 );
 
+const route = useRoute();
+const catalogue = route.params.catalogue || 'all';
```

Comment:
all ?

Reply:
'all' is the keyword for not limiting to a specific catalogue.